src/include/homestore/homestore.hpp

```diff
@@ -154,6 +153,7 @@ class HomeStore {
 
     // cap_attrs get_system_capacity() const; // Need to move this to homeblks/homeobj
     bool is_first_time_boot() const;
+    bool is_initializing() const { return !m_init_done; }
```

Comment:
This call is used by all subsystems/chunks instead of each of them maintaining is_inited flag and we need to walk through all chunks to set this after initialization. This is especially bad for Object storage since it can have 64K chunks.